packages/core/src/routes/connector.ts

```diff
@@ -122,66 +117,28 @@ export default function connectorRoutes<T extends AuthedRouter>(router: T) {
         })
       );
 
+      const insertConnectorId = generateConnectorId();
       ctx.body = await insertConnector({
-        id: generateConnectorId(),
+        id: insertConnectorId,
         ...body,
       });
 
-      return next();
-    }
-  );
-
-  router.patch(
-    '/connectors/:id/enabled',
-    koaGuard({
-      params: object({ id: string().min(1) }),
-      body: Connectors.createGuard.pick({ enabled: true }),
-    }),
-    async (ctx, next) => {
-      const {
-        params: { id },
-        body: { enabled },
-      } = ctx.guard;
-
-      const {
-        type,
-        dbEntry: { config },
-        metadata,
-        validateConfig,
-      } = await getLogtoConnectorById(id);
-
-      if (enabled) {
-        validateConfig(config);
-      }
-
-      // Only allow one enabled connector for SMS and Email.
-      // disable other connectors before enable this one.
-      if (enabled && (type === ConnectorType.Sms || type === ConnectorType.Email)) {
-        const connectors = await getLogtoConnectors();
+      if (
+        connectorFactory.type === ConnectorType.Sms ||
+        connectorFactory.type === ConnectorType.Email
+      ) {
+        const logtoConnectors = await getLogtoConnectors();
```

Comment:
can we keep the same logic for "Only allow one enabled connector for SMS and Email."? This PR is aim to remove "enabled", try to keep other things unchanged.